README.md

```diff
@@ -30,3 +30,15 @@ program that uses the ANTLR grammar for WDL.
 For some discussion comparing the two languages (mainly from the perspective of translating in the other direction, CWL to WDL), see this document:
 
 https://github.com/dnanexus/dxCompiler/blob/main/doc/CWL_v1.2.0_to_WDL_v1.md
+
+### Adding Test Cases
+
+1. Find a WDL workflow. Given below are some links that can be used to find these workflows: 
+    1. https://github.com/broadinstitute/warp
+    2. https://github.com/broadinstitute/viral-pipelines/tree/master/pipes/WDL
+    3. https://github.com/biowdl/tasks
+2. Use the translator to convert the WDL file and save the result to a .cwl file at a specified location. ```python wdl2cwl/main.py  path_to_wdl_file -o specified_location```.
+3. If a problem is encountered during the translation or if the WDL workflow has a feature that has not been implemented yet, submit a new issue with a description of the issue at https://github.com/common-workflow-lab/wdl-cwl-translator/issues
+4. Check whether the outputs of the WDL workflow and the resultant CWL file are equivalent upon giving the required inputs. The WDL workflow can be run using a workflow runner like miniwdl. (Refer the documentation https://github.com/chanzuckerberg/miniwdl) The CWL file can be run using cwltool (Refer the documentation https://github.com/common-workflow-language/cwltool)
```

Comment:
Okay, how do we get test data?

Reply:
Actually, I'm not quite sure about this. I haven't been very successful in finding the right files to run the workflows. Files I found through searching online don't always work.

Reply:
For tools like Picard they have great tests in the repository for the tool itself: https://github.com/broadinstitute/picard/search?q=CollectQualityYieldMetrics → https://github.com/broadinstitute/picard/blob/master/src/test/java/picard/analysis/CollectQualityYieldMetricsTest.java#L48 → https://github.com/broadinstitute/picard/blob/master/testdata/picard/sam/insert_size_metrics_test.sam

That file is small enough we could check it in to this repo (with attribution) and add an automated test that actually runs the input WDL and the resulting CWL description with this file